### PR TITLE
(Feature) Add Staff & User Ticket Notifications

### PR DESCRIPTION
**Pull request by Obi-Wana**
Adds Notifications (similar to new PM's) for new & Updated Tickets.
A new Migration has been created adding a "user_read" & "staff_read" column to the Ticket DB Table.

### CODE REVIEW

File: resources/views/partials/top_nav.blade.php [hunk 1]
```diff
@@ -58,9 +76,16 @@
                     </a>
                 </li>
             @endif
+
```
Comment (HDVinnie) [inline resources/views/partials/top_nav.blade.php:79]: Double check this PR please. Theres code that shouldn't belong.

**Comment by Obi-Wana** [comment]
@HDVinnie Removed the code that shouldn't belong into the top-nav.

File: database/migrations/2021_03_14_093812_add_read_column_tickets_table.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('tickets', function (Blueprint $table) {
+            $table->tinyInteger('user_read')->nullable()->after('staff_id');
```
Comment (HDVinnie) [inline database/migrations/2021_03_14_093812_add_read_column_tickets_table.php:17]: Would it not be more appropriate if these two new columns where a boolean with a default of 0 or false which would also allow you to get rid of these lines. https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/149f3a3c09dcac2cfbc2a0cc9f436e51ebca24b4/app/Http/Controllers/CommentController.php#L556-L561 ?

Comment (Obi-Wana) [reply database/migrations/2021_03_14_093812_add_read_column_tickets_table.php:17]: @HDVinnie I have used the table "private_messages" as template, where the "read" column is a tinyint. besides having less code, i dont see much of a performance improvement in using a bool with default 0.
However, I'm not that experienced yet. Decision's up to you.

**Pull request merged**